README.md

```diff
@@ -2,6 +2,10 @@
 
 This GitHub workflow action uses [Ziti NodeJS SDK](https://github.com/openziti/ziti-sdk-nodejs) to post an event's `payload` in JSON format over a `Ziti` connection.  
 
+## MacOS Compatibility
+
+If you have a MacOS job you may wish to use `@v1` ref which works with the `macos-latest` hosted runner.
```

Comment:
does this matter whatsoever? Why not let it die?

Reply:
Because someone may have a job that uses the MacOS runner, and so it would be inconvenient for them to maintain a separate job that uses the Linux runner.